Makefile

```
@@ -30,6 +30,7 @@ CLUSTER_NAME ?= test-gslb
 CLUSTER_GEO_TAGS ?= eu us cz af ru ap uk ca
 CHART ?= k8gb/k8gb
 CLUSTER_GSLB_NETWORK = k3d-action-bridge-network
+CLUSTER_GSLB_GATEWAY = $(shell docker network inspect ${CLUSTER_GSLB_NETWORK} | jq -r .[].IPAM.Config[0].Gateway)
```

this adds the dependency on `jq`, we should probably mention it somewhere otherwise it would fail silently. Well, `CLUSTER_GSLB_GATEWAY` would contain some err msg

---

Added check for `CLUSTER_GSLB_GATEWAY` emptynes

---

No need for `jq` dependendcy here, it can be solved with `docker` `--format` option, i.e.:
```suggestion
CLUSTER_GSLB_GATEWAY = $(shell docker network inspect ${CLUSTER_GSLB_NETWORK} -f '{{ (index .IPAM.Config 0).Gateway }}')
```